modules/cluster-logging-about.adoc

```diff
@@ -34,3 +34,21 @@ Custom Resource and adjusts the logging deployment accordingly.
 
 Administrators and application developers can view the logs of the projects for which they have view access.
 
+[NOTE]
+====
+If you are deploying cluster logging into a cluster that uses multitenant mode rather than the default network policy, you must create a network policy object in the `openshift-logging` namespace that allows ingress from the `openshift-operators-redhat` project to the `openshift-logging` project.
```

Comment:
@mburke5678 

So it's worth noting that, this pertains only to OpenShift SDN; If someone is using the OVN-Kubernetes network provider, there is no _mode_ as such as only network policy objects are supported.

So this should probably be something more like

> If you are deploying cluster logging into a cluster that uses the OpenShift SDN pod network provider in multitenant isolate mode...

But there's no network policy involved, instead it's `oc adm pod-network join-projects <a> <b>`.